### ISSUE TITLE
About labeled information in build_dataloader

### ISSUE THREAD

**Issue by HuKaiLang** [zeludeng/insclr#4]
Hi, all dataloaders produced by the `build_dataloader` method under the `insclr/benchmark/data/__init__.py` file seem to use Label information, which doesn't seem to fit the concept of self-supervision in paper? Self-supervised learning should not be able to use manually labeled information, right?

reference code:
```
if dataset == "gldv2":
    dataset = GLDv2(
        root="datasets/gldv2", fname="train_clean.csv",
        transform=train_transform, target_transform=None, noaug_transform=noaug_transform
    )
else:
    dataset = BaseDataset(
        cfg.DATA.ROOT, fname=cfg.DATA.TRAIN_FILENAME,
        transform=train_transform, target_transform=None, noaug_transform=noaug_transform
    )
train_dataset = DatasetWithNeighbors(dataset, neighbors, cfg.NEIGHBOR.K)
```

If that's the case, the comparison of your work with methods such as MoCoV2, SimCLR, etc. is not fair.
